params/protocol_params.go

```diff
@@ -153,4 +153,6 @@ var (
 	LighthouseDurationLimit        = big.NewInt(7)     // The decision boundary on the blocktime duration used to determine whether difficulty should go up or not.
 	LocalDurationLimit             = big.NewInt(2)     // The decision boundary on the blocktime duration used to determine whether difficulty should go up or not.
 	TimeFactor                     = big.NewInt(7)
+	PrimeEntropyTarget             = big.NewInt(441) // This is TimeFactor*TimeFactor*common.NumZonesInRegion*common.NumRegionsInPrime
```

Comment:
This would be more readable/easier to maintain

```PrimeEntropyTarget := new(big.Int).Mul(timeFactor, timeFactor).Mul(common.NumZonesInRegion).Mul(common.NumRegionsInPrime)```